Lab2.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lab2</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" >
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="menu-caption lower-menu">Navigation</div>
+            <div class="menu">
+                <i class="fas fa-bars menu-icon lower-menu"></i>
+                <ul>
+                    <li><a href="#">Psd websites</a></li>
```

Comment:
It's okay, but could you please expand a tag clickable area to make it possible to click on li element?